packages/di/src/lib/injectable-el.ts

```diff
@@ -5,45 +5,49 @@ export function injectableEl<T extends ConstructableToken<HTMLElement>>(
   Base: T,
   _ctx: ClassDecoratorContext
 ) {
-  return class InjectablElement extends Base {
-    constructor(..._: any[]) {
-      super();
+  const def = {
+    [Base.name]: class extends Base {
```

Comment:
NOTE: Add comment to explain why

Reply:
this looks indeed very unusual, but it works?

need to give this a test... sadly I forgot what I was debugging back then xD

Reply:
Yeah I need to add comments to explain. But anonymous classes take their name from the variable they are assigned to. If I create an object first I can dynamically assign the name.

Reply:
I completely forgot about this when we talked about it last